README.md

```diff
@@ -4,6 +4,8 @@ This repository serves as a template for creating a new Looker Extension.
 
 It uses [React](https://reactjs.org/) and [TypeScript](https://www.typescriptlang.org/) for writing your extension, the [React Extension SDK](https://github.com/looker-open-source/extension-sdk-react) for interacting with Looker, and [Webpack](https://webpack.js.org/) for building your code.
 
+__Important:__ In general the template targets Looker version 7 and above. Should you wish to build an extension for Looker 6.24, please use branch `looker-6.24-extension-sdk-0.3.2`.
```

Comment:
I just think 7 should be the minimum regardless.

Reply:
I think the issue is that Lindsay/Delaney wanted to say that the closed beta was available. Unfortunately Looker 6.24 has a couple of defects that won't work with the extension SDK. The patched version of the extension SDK referenced in the branch allows it to work with Looker 6.24. I guess it's all semantics. We could say that the closed beta is available but only with a Looker instance that is built from master. If you, Lindsay and Delaney are okay with that then we don't need the branch.